tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -65,3 +64,45 @@ def test_n_flag(tmp_path, simple_scenes_path):
     ]
     _, err, exit_code = capture(command)
     assert exit_code == 0, err
+
+
+def test_quality_flags():
```

Comment:
Tests are perfect but not at the right place; as your tests does not involve any video rendering.  

Move it to tests/test_cli_flags.py (or another name, idk). 

Maybe we should merge scene rendering tests and other tests, idk

Reply:
Done!